sdk/mobile-center-updates/src/test/java/com/microsoft/azure/mobile/updates/UpdatesBeforeDownloadTest.java

```diff
@@ -64,14 +64,13 @@ public ServiceCall answer(InvocationOnMock invocation) throws Throwable {
         ReleaseDetails releaseDetails = mock(ReleaseDetails.class);
         when(releaseDetails.getId()).thenReturn("someId");
         when(ReleaseDetails.parse(anyString())).thenReturn(releaseDetails);
-        Context context = mock(Context.class);
-        when(context.getPackageName()).thenReturn("com.contoso");
-        PackageManager packageManager = mock(PackageManager.class);
-        when(context.getPackageManager()).thenReturn(packageManager);
-        when(packageManager.getPackageInfo("com.contoso", 0)).thenThrow(new PackageManager.NameNotFoundException());
+
+        /* Override mock answer by calling it once then configuring new answer. */
+        mPackageManager.getPackageInfo("com.contoso", 0);
```

Comment:
There is an answer configured in AbstractUpdateTests.setUp, when you configure two when rules, the firs when needs to be called once, the second when clause will be used for further answers.